script/cibuild.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+set -e # halt script on error
+
```

Comment:
Travis does the `bundle install` by default. Also, I'd prefer the `bundle install` step to stay separate from test running.

It feels weird to me to have the test runner actually disturb the code environment. Too much magic.